cinder/volume/drivers/lightos.py

```diff
@@ -1062,23 +1066,39 @@ def __add_volume_acl(self, project_name, lightos_volname, acl_to_add):
             LOG.warning('Got LightOS volume without ACL?! data: %s', data)
             return False
 
+        ip_acl= data.get('IPAcl')         
+        if not ip_acl:
+           LOG.warning('Got LightOS volume without IP ACL?! data: %s', data)
+           return False
+
         acl = acl.get('values', [])
+        ip_acl = ip_acl.get('values', [])
+        
 
         # remove ALLOW_NONE and add our acl_to_add if not already there
         if 'ALLOW_NONE' in acl:
             acl.remove('ALLOW_NONE')
         if acl_to_add not in acl:
             acl.append(acl_to_add)
 
+        if 'ALLOW_ANY' in ip_acl:
+            ip_acl.remove('ALLOW_ANY')
+        
+        if 'ALLOW_NONE' in ip_acl:
+            ip_acl.remove('ALLOW_NONE')
+        
+        ip_acl=set(ip_acl).union(set(host_ips))
```

Comment:
I don't think we want to enable this behavior (IPACL) unconditionally. Better to add Cinder.conf configuration option `lightos_use_ipacl`, which can default to disabled (False), and once we are certain it's working, we'll make the default enabled (True).